joint-discovery-gatk4.wdl

```diff
@@ -595,21 +596,21 @@ task SNPsVariantRecalibratorCreateModel {
       -resource dbsnp,known=true,training=false,truth=false,prior=7:${dbsnp_resource_vcf}
   }
   runtime {
-    docker: docker_image
-    memory: mem_size
+    docker: docker
+    memory: "104 GB"
```

Comment:
This is also a lot higher than it needs to be.

Reply:
I guess this is on the is_large_callset code path, so it needs more memory than the standard SNP VariantRecalibrator.  Again, most users won't have that many samples so it's not a huge issue.